### DISCUSSION TITLE
iOS Support

### DISCUSSION THREAD

**Discussion by p1337** [markqvist/Reticulum#644] [Ideas]
Is there any option to connect an iPhone to the Heltec Lora32 V3 ? 
I followed instructions here : for the iOS Beta Test -> but couldn't get it running:

is there some developement for iOS on going ?

**Comment by liamcottle** [comment]
There's another discussion over here you might be interested in:
https://github.com/markqvist/Reticulum/discussions/307#discussioncomment-11690826
